tools/install.sh

```diff
@@ -43,19 +62,25 @@ mv -f ~/.zshrc-omztemp ~/.zshrc
 
 TEST_CURRENT_SHELL=$(expr "$SHELL" : '.*/\(.*\)')
 if [ "$TEST_CURRENT_SHELL" != "zsh" ]; then
-    echo "\033[0;34mTime to change your default shell to zsh!\033[0m"
+    printf "${BLUE}Time to change your default shell to zsh!${NORMAL}\n"
     chsh -s $(grep /zsh$ /etc/shells | tail -1)
 fi
 unset TEST_CURRENT_SHELL
 
-echo "\033[0;32m"'         __                                     __   '"\033[0m"
-echo "\033[0;32m"'  ____  / /_     ____ ___  __  __   ____  _____/ /_  '"\033[0m"
-echo "\033[0;32m"' / __ \/ __ \   / __ `__ \/ / / /  /_  / / ___/ __ \ '"\033[0m"
-echo "\033[0;32m"'/ /_/ / / / /  / / / / / / /_/ /    / /_(__  ) / / / '"\033[0m"
-echo "\033[0;32m"'\____/_/ /_/  /_/ /_/ /_/\__, /    /___/____/_/ /_/  '"\033[0m"
-echo "\033[0;32m"'                        /____/                       ....is now installed!'"\033[0m"
-echo "\n\n \033[0;32mPlease look over the ~/.zshrc file to select plugins, themes, and options.\033[0m"
-echo "\n\n \033[0;32mp.s. Follow us at http://twitter.com/ohmyzsh.\033[0m"
-echo "\n\n \033[0;32mp.p.s. Get stickers and t-shirts at http://shop.planetargon.com.\033[0m"
+printf "${GREEN}"
+echo '         __                                     __   '
+echo '  ____  / /_     ____ ___  __  __   ____  _____/ /_  '
+echo ' / __ \/ __ \   / __ `__ \/ / / /  /_  / / ___/ __ \ '
+echo '/ /_/ / / / /  / / / / / / /_/ /    / /_(__  ) / / / '
+echo '\____/_/ /_/  /_/ /_/ /_/\__, /    /___/____/_/ /_/  '
+echo '                        /____/                       ....is now installed!'
+echo ''
+echo ''
+echo 'Please look over the ~/.zshrc file to select plugins, themes, and options.'
+echo ''
+echo 'p.s. Follow us at https://twitter.com/ohmyzsh.'
+echo ''
+echo 'p.p.s. Get stickers and t-shirts at http://shop.planetargon.com.'
+echo ''
+printf "${NORMAL}"
 env zsh
```

Comment:
Actually the `. ~/.zshrc` was wrong as well with the previous method of installation (`curl | sh`), since it would be read by `sh` and possibly lead to syntax errors.

As is now, here's the behavior we get:

##### `env zsh`:
1. A subshell is created running zsh.
2. User finishes session, runs `exit` or `CTRL`+`D`.
3. Goes back to `sh -c "$(wget)"`.
4. Has no more instructions to run, the `sh` process finishes.
5. Back to initial shell where the user ran `sh -c ...`

##### `exec zsh`:
1. The current `sh -c ...` process is substituted by `zsh`
2. User finishes session, runs `exit` or `CTRL`+`D`.
3. Back to initial shell where the user ran `sh -c ...`

So, in terms of behavior, both are equally flawed, and I think this is as good a behavior as we're gonna get.

Regarding executing directly or printing a message, I think it's ok that we're at least showing what they'll get once they restart the terminal / [logout and log back in](https://github.com/robbyrussell/oh-my-zsh/issues/4298). Perhaps we should make it explicit that they need to restart/logout once they're done? 

I don't know, every time I try to think in terms of what the user might do or think I fail miserably. It's probably good as it is now.